src/commands/getAddon.ts

```diff
@@ -20,13 +20,16 @@ export async function getInput(): Promise<string> {
   return input;
 }
 
-export async function downloadAndExtract() {
+export async function downloadAndExtract(
+  urlGuid?: string,
+  urlVersion?: string
+) {
```

Comment:
can we have a new test for `downloadAndExtract` that covers the parameters being present?

Reply:
We don't have any tests for downloadAndExtract right now (and I'm not sure how to test it since it's just a string of function calls)

Reply:
I was thinking a test that confirmed the `urlGuid` was used if provided, rather than `getInput`.  Can you mock some of the functions to check if `getInput` was called and `getVersionChoice` is called with the correct value?